modules/virt-removing-mediated-device-from-cluster-cli.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-removing-mediated-device-from-cluster_{context}"]
+= Removing mediated devices from the cluster
+
+To remove a mediated device from the cluster, delete the information for that device from the `HyperConverged` custom resource (CR).
+
+.Procedure
+
+. Edit the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Remove the device information from the `spec.permittedHostDevices.mediatedDevices` array by deleting the `mdevNameSelector` and `resourceName` fields for that device. For example:
++
+.Example configuration file
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  mediatedDevicesConfiguration:
+    mediatedDevicesTypes:
+      - nvidia-231
```

Comment:
We should also mention to remove these entries in the doc.

As, this section, `creates the mdev device` and the below one `only exposes the mdev devices to the cluster.`

Reply:
Only by dropping both the configs, we would be able to then configure `another mdev device type`